src/entity.js

```diff
@@ -1,5 +1,4 @@
-"use strict";
-
+/* eslint-disable consistent-this */
```

Comment:
Discussed in irc -- this file's use of subclassing is unusual, but correct and effective.